pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -227,6 +228,12 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 		return nil, err
 	}
 	capacity = cadvisor.CapacityFromMachineInfo(machineInfo)
+	pidlimits, err := pidlimit.Stats()
+	if err == nil && pidlimits != nil && pidlimits.MaxPID != nil {
+		capacity[pidlimit.PIDs] = *resource.NewQuantity(
```

Comment:
the node status update calls `GetCapacity()` at the bottom of this file to populate the node status, which returns cm.capacity.  We either need to filter out pids when we return the capacity there, or need to store it somewhere else in the container manager.

Reply:
Perhaps a new getAllCapacity() (since the cgroup manager is also in the cm package) that returns everything, including any other "resources" that we don't want to make public?

Reply:
That would be a reasonable approach.  Or having getInternalCapacity and GetExternalCapacity might make it more clear.  But whatever you think is most readable is good with me.

Reply:
@dashpole is the current code OK now?